pkg/ops/dataServerOps.go

```diff
@@ -19,31 +20,31 @@ type DataServerOps struct {
 	exitCh  chan os.Signal
 	done    chan int
 	server  *dataServer.DataServer
-	log     *util.Logger
+	log     log.Logger
 	Running bool
 }
 
 // CreateDataServerOps creates a data server instance for runtime.
-func CreateDataServerOps(ctx context.Context, exitCh chan os.Signal) (*DataServerOps, error) {
-	ds, err := dataServer.CreateServer(ctx)
+func CreateDataServerOps(ctx context.Context, logger log.Logger, exitCh chan os.Signal) (*DataServerOps, error) {
+	ds, err := dataServer.CreateServer(ctx, log.With(logger, "component", "ops"))
 	if err != nil {
 		return nil, err
 	}
 	done := make(chan int)
-	ops := &DataServerOps{exitCh: exitCh, server: ds, log: util.NewLogger("ops", "DataServerOps"), done: done, Running: false}
+	ops := &DataServerOps{exitCh: exitCh, server: ds, log: logger, done: done, Running: false}
 
 	return ops, nil
 }
 
 // Start the data server.
 func (ops *DataServerOps) Start(ctx context.Context) error {
-	if err := ops.server.Start(ctx, ops.done); err != nil {
+	if err := ops.server.Start(ctx, ops.log, ops.done); err != nil {
```

Comment:
the logger was already added to the `ops.server` with the `dataServer.CreateServer` constructor.